public/elements/zero-featured-guardian.html

```diff
@@ -121,7 +121,7 @@
       },
 
       _toggle: function() {
-        var moreInfo = document.getElementById('guardian-qa');
+        var moreInfo = document.getElementById('guardian-qa-' + this.guardian);
```

Comment:
Yea this was to fix a bug where adding more than one past guardian, the IDs were not unique.  So toggling one section opened them all up.